src/main/java/com/cloudbees/plugins/deployer/DeployListener.java

```diff
@@ -117,11 +117,14 @@ public static void notifySuccess(DeployEvent event) {
             @CheckForNull DeployHost<S, T> host,
             @CheckForNull DeployTarget<T> target,
             @NonNull DeployEvent event) {
-        for (DeployListener listener : Jenkins.getInstance().getExtensionList(DeployListener.class)) {
-            try {
-                listener.onSuccess(host, target, event);
-            } catch (Throwable t) {
-                LOGGER.log(Level.WARNING, "Uncaught exception from " + listener.getClass(), t);
+        final Jenkins jenkins = Jenkins.getInstance();
+        if (jenkins != null) {
```

Comment:
Why? This way it will behave the same as is recommended after 1.575 with ```ExtensionList.lookup()``` i.e. it would iterate over ```0``` elements.